x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/rule_monitoring/logic/rule_execution_log/event_log/aggregations/execution_results/index.ts

```diff
@@ -330,10 +344,7 @@ export const formatAggExecutionEventFromBucket = (
     es_search_duration_ms: bucket?.ruleExecution?.esSearchDuration?.value ?? 0,
     schedule_delay_ms: scheduleDelayUs / ONE_MILLISECOND_AS_NANOSECONDS,
     timed_out: timedOut,
-    // security fields
-    indexing_duration_ms: bucket?.securityMetrics?.indexDuration?.value ?? 0,
-    search_duration_ms: bucket?.securityMetrics?.searchDuration?.value ?? 0,
-    gap_duration_s: bucket?.securityMetrics?.gapDuration?.value ?? 0,
+    ...mapSecurityFieldsFromExecutionBucket(bucket),
```

Comment:
nit: wouldn't `security_status` and `security_message` also be "security fields"?

Reply:
Nice catch! I have moved those fields to the mapping function.